### PR TITLE
Added textbook page

### PR DESCRIPTION
**Pull request by SarthakSingh31**

### CODE REVIEW

**Review by charlon — APPROVED** [review]

File: myuw/static/vue/components/textbooks/textbooks.vue [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    }),
+    ...mapGetters('textbooks', {
+      isTextbookReady: 'isReadyTagged',
+      isTextbookErrored: 'isErroredTagged',
+      statusCodeTextbooks: 'statusCodeTagged',
+      getProcessedData: 'getProcessedData',
+    }),
+    isReady() {
+      return (
+        this.isStudScheduleReady(this.term) &&
```
Comment (fanglinfang) [inline myuw/static/vue/components/textbooks/textbooks.vue:112]: What if this user is an instructor, not a student?

Comment (SarthakSingh31) [reply myuw/static/vue/components/textbooks/textbooks.vue:112]: https://github.com/uw-it-aca/myuw/blob/7f20bf56a3f3f3f1b791b3acb468e06a64fd14df/myuw/static/vue/components/textbooks/textbooks.vue#L114-L117

This value is false when the inst data is still fetching, so it wait for a response from the inst api.

Comment (fanglinfang) [reply myuw/static/vue/components/textbooks/textbooks.vue:112]: should we have is_student && isStudScheduleReady || is_instructor && isInstScheduleReady ?

Comment (SarthakSingh31) [reply myuw/static/vue/components/textbooks/textbooks.vue:112]: we need both of them to be fetched before loading the page, I will add a conditional on the isStudScheduleReady as well

**Review by fanglinfang — APPROVED** [review]

**Pull request merged**